images/router/haproxy/conf/haproxy-config.template

```diff
@@ -289,7 +289,7 @@ backend be_tcp_{{$cfgIdx}}
     {{ end }}
     {{ with $value := index $cfg.Annotations "haproxy.router.openshift.io/timeout"}}
       {{if (matchPattern "[1-9][0-9]*(us|ms|s|m|h|d)?" $value) }}
-  timeout server  {{$value}}
+  timeout tunnel  {{$value}}
```

Comment:
Hmm, `timeout tunnel` does have some implications for websockets - if its too low, websockets/upgraded connections would timeout often. In this specific case, since we are setting it only if there is an annotation, its ok. 
But its worth noting this will override `timeout tunnel 1h` set in the `defaults` section.  Maybe worth adding to the docs.